runtime/onert/core/src/util/shapeinf/Select.cc

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "util/ShapeInference.h"
+
+namespace onert
+{
+namespace shape_inference
+{
+
+ir::Shape inferSelectShape(const ir::Shape &input_true_shape, const ir::Shape &input_false_shape)
+{
+  if (input_true_shape != input_false_shape)
+  {
+    throw std::runtime_error("All shapes must be the same.");
```

Comment:
Why should it be the same?
You should check more....

Reply:
I will modify